src/internet_identity/src/storage.rs

```diff
@@ -70,10 +71,8 @@
 //! In order to keep state across upgrades that is not related to specific anchors (such as archive
 //! information) Internet Identity will serialize the [PersistentState] on upgrade and restore it
 //! again after the upgrade.
```

Comment:
nit: the statement about serialisation and restoration is not needed anymore, and is somewhat misleading, maybe just combine with the the sentence on line 74, e.g. "Internet Identity maintains a [PersistentState], which is stored in ..."

Reply:
Thanks. I still added another sentence to mention the `pre_upgrade` hook.